sc-memory/sc-memory/sc_iterator.cpp

```diff
@@ -18,7 +18,7 @@ sc_iterator3 * CreateIterator3<sc_addr, sc_type, sc_addr>(
 }
 
 template <>
-sc_iterator3 * CreateIterator3<sc_addr, sc_type, sc_type>(
+sc_iterator3 * GenerateIterator3<sc_addr, sc_type, sc_type>(
```

Comment:
'Generate' has meaning 'generate in knowledge base'. Iterator doesn't have representation in KB